app/src/main/java/se/inera/intyg/minaintyg/error/service/dto/LogErrorRequest.java

```diff
@@ -0,0 +1,11 @@
+package se.inera.intyg.minaintyg.error.service.dto;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class LogErrorRequest {
+
+  ErrorData error;
```

Comment:
Do we need to wrap the error in its own request? Could the members of ErrorData be part of the LogErrorRequest?